README.md

```diff
@@ -1,350 +1,57 @@
-      console.log('sent message', message);
-      setMessage(''); // Clear input after successful send
-    } catch (error) {
-      console.error('error sending message', error);
-    }
-  };
+---
 
-  // This is a very simple UI that displays the messages and a text input for sending messages.
-  return (
-    <div style={{
-      maxWidth: '600px',
-      minWidth: '400px',
-      margin: '20px auto',
-      fontFamily: 'Arial, sans-serif'
-    }}>
-      {/* Container for the messages */}
-      <div className="messages-container" style={{
-        height: '400px',
-        overflowY: 'auto',
-        border: '1px solid #e0e0e0',
-        borderRadius: '8px',
-        marginBottom: '20px',
-        padding: '16px',
-        backgroundColor: '#f8f9fa'
-      }}>
-        {messages.map((msg, index) => (
-          <div
-            key={index}
-            className="message"
-            style={{
-              backgroundColor: 'white',
-              padding: '10px 15px',
-              borderRadius: '12px',
-              marginBottom: '8px',
-              boxShadow: '0 1px 2px rgba(0,0,0,0.1)',
-              maxWidth: '80%'
-            }}
-          >
-            {/* Display the message timestamp and text */}
-            <div style={{ fontSize: '0.8em', color: '#666', marginBottom: '4px' }}>
-              {new Date(msg.timestamp).toLocaleTimeString()}
-            </div>
-            <div style={{ wordBreak: 'break-word', color: '#333' }}>
-              {msg.text}
-            </div>
-          </div>
-        ))}
-      </div>
-      <div className="input-container" style={{
-        display: 'flex',
-        gap: '10px'
-      }}>
-        {/* Input field for sending messages */}
-        <input
-          type="text"
-          value={message}
-          onChange={(e) => setMessage(e.target.value)}
-          placeholder="Type your message..."
-          style={{
-            flex: 1,
-            padding: '10px',
-            borderRadius: '4px',
-            border: '1px solid #e0e0e0',
-            fontSize: '16px'
-          }}
-          onKeyDown={(e) => {
-            if (e.key === 'Enter' && !e.shiftKey) {
-              e.preventDefault();
-              handleSend();
-            }
-          }}
-        />
-        {/* Button for sending messages */}
-        <button
-          onClick={handleSend}
-          style={{
-            padding: '10px 20px',
-            backgroundColor: '#007bff',
-            color: 'white',
-            border: 'none',
-            borderRadius: '4px',
-            cursor: 'pointer',
-            fontSize: '16px'
-          }}
-        >
-          Send
-        </button>
-      </div>
-    </div>
-  );
-}
-```
+## Getting started
 
-Now, update your `App.tsx` (or similar) to initialize the Ably Chat client and render the component you just created. Replace the `<API_KEY>` with your
-own key from the Ably dashboard.
+Everything you need to get started with Ably:
```

Comment:
Should we have an explicit callout of the docs here? Appreciate that the getting started's are on the docs page, but I could imagine people may be looking for a link that says "read the docs"